assets/js-translations/course_staff.fi.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
No need to rename the file, even though the original file isn't perfectly named.